landoui/templates/partials/layout.html

```diff
@@ -13,7 +13,7 @@
 {% include "partials/navbar.html" %}
 <section class="hero is-light is-bold">
   <div class="container">
-    <p class="subtitle">{% block banner_text %}{% endblock %}</p>
+    <div class="subtitle"></div>
```

Comment:
Do you not like the little banner? It is redundant currently on the sign out page, but, I feel it adds some character to the site and could be useful once the signout page has more information. 

Also if you wanted to remove it you have to remove it from all the other pages.